app/command.go

```diff
@@ -530,7 +530,7 @@ func (a *App) HandleCommandResponse(command *model.Command, args *model.CommandA
 	_, err := a.HandleCommandResponsePost(command, args, response, builtIn)
 
 	if err != nil {
-		mlog.Error("error occurred in handling command response post", mlog.Err(err))
+		mlog.Debug("error occurred in handling command response post", mlog.Err(err))
```

Comment:
Let's make log lines to a proper statement by starting with a capital letter and ending in a period.